src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -252,6 +252,8 @@
   "text.autoconfig.skyblocker.option.locations.dungeons.dungeonChestProfit.lossColor": "Loss Color",
   "text.autoconfig.skyblocker.option.locations.dungeons.croesusHelper": "Croesus Helper",
   "text.autoconfig.skyblocker.option.locations.dungeons.croesusHelper.@Tooltip": "Gray out chests that have already been opened.",
+  "text.autoconfig.skyblocker.option.locations.dungeons.croesusProfit": "Croesus Chest Profit Calculator",
+  "text.autoconfig.skyblocker.option.locations.dungeons.croesusProfit.@Tooltip": "Highlight chests with best profit at Croesus NPC.\n\n\nBest chest will be highlighted green.\nIf there is a chest worth using dungeon key on, it will be highlighted yellow",
```

Comment:
```json
"text.autoconfig.skyblocker.option.locations.dungeons.croesusProfit.@Tooltip": "Highlights chests with the best profit at the Croesus NPC.\n\n\nThe best chest will be highlighted green.\nIf there is a chest worth using dungeon key on, it will be highlighted yellow",
```